lib/compile_time.flow

```diff
@@ -0,0 +1,20 @@
+export {
+	// Speculative function, used to make output of computed constant values during compilation.
+	// Normally these function is eliminated from program before passing it to backends.
+	outputCt(msg: string, v: ?) -> ? {
+		v
+	}
+
+	// Speculative function, used to make forbid a compile-time computation of an exression.
+	// Normally these function is eliminated from program before passing it to backends.
+	forbidCt(v: ?) -> ? {
```

Comment:
I don't see what the point of this one is.

Reply:
We need this for example for JAVA: when some generated array is more than 64K, we have to split it into parts, we do not want the compiler to concatenate these arrays back.